google/cloud/bigtable/internal/table_admin.h

```diff
@@ -477,6 +477,62 @@ class TableAdmin {
         std::forward<Functor>(callback));
     retry->Start(cq);
   }
+
+  /**
+   * Make an asynchronous request to delete a snapshot.
+   *
+   * @warning This is a private alpha release of Cloud Bigtable snapshots. This
+   * feature is not currently available to most Cloud Bigtable customers. This
+   * feature might be changed in backward-incompatible ways and is not
+   * recommended for production use. It is not subject to any SLA or deprecation
+   * policy.
+   *
+   * @param cluster_id the cluster id to which snapshot is associated.
+   * @param snapshot_id the id of the snapshot.
+   * @param cq the completion queue that will execute the asynchronous calls,
+   *     the application must ensure that one or more threads are blocked on
+   *     `cq.Run()`.
+   * @param callback a functor to be called when the operation completes. It
+   *     must satisfy (using C++17 types):
+   *     static_assert(std::is_invocable_v<
+   *         Functor, google::protobuf::Empty&,
+   *         grpc::Status const&>);
+   *
+   * @tparam Functor the type of the callback.
+   */
```

Comment:
Please add a `TODO(#1325) - eliminate usage of google::protobuf::Empty in the callback` somewhere in the comment block.